Formula/math-comp.rb

```diff
@@ -0,0 +1,52 @@
+class MathComp < Formula
+  desc "Mathematical Components for the Coq proof assistant"
+  homepage "https://math-comp.github.io/math-comp/"
+  url "https://github.com/math-comp/math-comp/archive/mathcomp-1.6.4.tar.gz"
+  sha256 "c672a4237f708b5f03f1feed9de37f98ef5c331819047e1f71b5762dcd92b262"
+  head "https://github.com/math-comp/math-comp.git"
+
+  depends_on "ocaml" => :build
+  depends_on "coq"
+
+  def install
+    coqbin = "#{Formula["coq"].opt_bin}/"
+    coqlib = "#{lib}/coq/"
+
+    (buildpath/"mathcomp/Makefile.coq.local").write <<~EOS
+      COQLIB=#{coqlib}
+    EOS
+
+    cd "mathcomp" do
+      system "make", "MAKEFLAGS=#{ENV["MAKEFLAGS"]}", "COQBIN=#{coqbin}"
+      system "make", "install"
```

Comment:
do we not need to pass `"MAKEFLAGS=#{ENV["MAKEFLAGS"]}", "COQBIN=#{coqbin}"` on this line too?

Reply:
Also do `make` and `make install` really need to be separate steps?

Reply:
> Also do make and make install really need to be separate steps

Yes. No dependency is defined on the default target (`all`) and `install`, and so they must be invoked separately.

Reply:
And do we not need to pass `"MAKEFLAGS=#{ENV["MAKEFLAGS"]}", "COQBIN=#{coqbin}"` here?